nncf/experimental/onnx/graph/model_transformer.py

```diff
@@ -306,3 +323,38 @@ def _insert_quantizer_dequantizer(self, transformation: ONNXQuantizerInsertionCo
         insert_index = onnx_graph.get_node_index(input_nodes[0].name)
         self._model.graph.node.insert(insert_index, quantizer)
         self._model.graph.node.insert(insert_index + 1, dequantizer)
+
+    def _apply_bias_correction_transformations(self, transformations: List[ONNXBiasCorrectionCommand]) -> None:
+        """
+        Applies bias correction transformations on the model
+
+        :param transformations: lisf of the bias correction transformations
+        """
+        onnx_graph = ONNXGraph(self._model)
+
+        for transformation in transformations:
+            node_name = transformation.target_point.target_node_name
+            onnx_node = onnx_graph.get_node_by_name(node_name)
+            bias_initializer_name = onnx_node.input[2]
+            bias_initializer = onnx_graph.get_initializer(bias_initializer_name)
+            current_bias_value = onnx.numpy_helper.to_array(bias_initializer)
+
+            new_bias_value = current_bias_value + transformation.bias_shift
```

Comment:
Sharing such logic is a problem we haven't addressed yet. The only idea I have is to implement basic math operations and helpers (like .copy(), .from_array(), etc.) in NNCFTensor. But it should bring a significant burden to support and develop this. Anyway, let's keep it out of the scope of this PR.

Reply:
Replaced this logic into the algorithm.